### PR TITLE
update libsodium install to install version 1.0.12

### PR DESCRIPTION
**Pull request by deerdear**
Signed-off-by: mountbranch <jonatan@c8.works>

### CODE REVIEW

File: doc/mac-build.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
    ```
    brew install pkg-config
-   brew install libsodium
+   brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/65effd2b617bade68a8a2c5b39e1c3089cc0e945/Formula/libsodium.rb
```
Comment (jovfer) [inline doc/mac-build.md]: Please add comment about reason to set libsodium from url. (non-last 1.0.12 version required)

**Comment by jovfer** [comment]
@deerdear DCO check is failed for the first commit in PR. You can use interactive rebase or cherry-pick to fix it.

**Comment by jovfer** [comment]
@deerdear DCO still failed. Sign-off should presents in all commits of the PR. Please rebase or cherry-pick useful commits. After that you can force push to this branch or create new one PR.

**Comment by jovfer** [comment]
@deerdear could you please fix DCO for first 2 commits and force-push to this PR branch?
I suggest to use interactive rebase
```
git fetch --all
git checkout patch-2
git rebase -i hyperledger/master (use reword for each commit and manually add correct string for sign)
git push -f
```

**Comment by jovfer** [comment]
For all 3 useful commits signature should be 
```
 Signed-off-by: Jonatan Bergquist <jonatan.h.bergquist@gmail.com>
```

**Comment by deerdear** [comment]
Hi @jovfer  Thanks, I'm trying but I don't really get what to do. I'm trying to do `git rebase -i hyperledger/master` but I get this error then:`fatal: Needed a single revision
invalid upstream hyperledger/master`

**Comment by jovfer** [comment]
@deerdear did you have link to this remote (hyperledger) in your local git repo copy?

**Comment by deerdear** [comment]
@jovfer No, how do I add it?

**Comment by jovfer** [comment]
@deerdear 'git remote add https://github.com/hyperledger/indy-sdk.git'

**Comment by deerdear** [comment]
No, not working. I did `git remote add hyperledger/master https://github.com/hyperledger/indy-sdk.git`

**Comment by jovfer** [comment]
@deerdear let's continue in rocketchat.

**Comment by vimmerru** [comment]
@deerdear May be just issue create new branch from our master, add this small fix, commit with -s and issue new PR?

**Comment by jovfer** [comment]
@vimmerru there are some links to this PR. I prefer to just fix this one

**Pull request merged**